README/MATH/Prediction_Market_Math.tex

```diff
@@ -0,0 +1,1763 @@
+
+\subsection*{Aggregation Equation}
+
+For each user \(\text{User}_u\), the total shares for YES and NO outcomes are calculated as follows:
+
+\[
+\text{YesSharesOwned}_u = \sum_{\substack{i \\ \text{Username}_i = \text{User}_u \\ \text{Outcome}_i = \text{YES}}} P_i
+\]
+
+\[
+\text{NoSharesOwned}_u = \sum_{\substack{i \\ \text{Username}_i = \text{User}_u \\ \text{Outcome}_i = \text{NO}}} P_i
+\]
+
+\subsection*{Negative Share Adjustment}
+
+If any aggregated share count is negative, it is adjusted to zero:
+
+\[
+\text{YesSharesOwned}_u =
+\begin{cases}
+\text{YesSharesOwned}_u & \text{if } \text{YesSharesOwned}_u \geq 0 \\
+0 & \text{if } \text{YesSharesOwned}_u < 0
+\end{cases}
+\]
+
+\[
+\text{NoSharesOwned}_u =
+\begin{cases}
+\text{NoSharesOwned}_u & \text{if } \text{NoSharesOwned}_u \geq 0 \\
+0 & \text{if } \text{NoSharesOwned}_u < 0
+\end{cases}
+\]
+
+\subsection*{Output}
+
+The final output is a set of market positions:
+
+\[
+\text{Positions} = \{ \text{MarketPosition}_u \,|\, u = 1, 2, \dots, m \}
+\]
+
+where each market position is represented as:
+
+\[
+\text{MarketPosition}_u = (\text{Username}_u, \text{YesSharesOwned}_u, \text{NoSharesOwned}_u)
+\]
+
+\section*{Normalize Market Positions}
+
+The normalization of market positions ensures that for each user, only one of \(\text{YesSharesOwned}\) or \(\text{NoSharesOwned}\) is greater than zero, with the other being set to zero. The resulting value represents the net difference between the two.  This is in line with the convention that users can only hold YES or NO shares but not both simultaneously on the same market.
+
+\subsection*{Input Definitions}
+
+\begin{itemize}
+    \item \(\text{Positions} = \{ \text{MarketPosition}_u \,|\, u = 1, 2, \dots, m \}\): A set of market positions, where each position \(\text{MarketPosition}_u\) has the following attributes:
+    \begin{itemize}
+        \item \(\text{Username}_u\): The username of the user.
+        \item \(\text{YesSharesOwned}_u\): The total YES shares owned by the user.
+        \item \(\text{NoSharesOwned}_u\): The total NO shares owned by the user.
+    \end{itemize}
+\end{itemize}
+
+\subsection*{Normalization Equation}
+
+For each user \(\text{User}_u\), the normalized market position is calculated as:
+
+\[
+\text{If } \text{YesSharesOwned}_u > \text{NoSharesOwned}_u:
+\]
+\[
+\text{YesSharesOwned}_u^\text{Normalized} = \text{YesSharesOwned}_u - \text{NoSharesOwned}_u
+\]
+\[
+\text{NoSharesOwned}_u^\text{Normalized} = 0
+\]
+
+\[
+\text{If } \text{NoSharesOwned}_u \geq \text{YesSharesOwned}_u:
+\]
+\[
+\text{NoSharesOwned}_u^\text{Normalized} = \text{NoSharesOwned}_u - \text{YesSharesOwned}_u
+\]
+\[
+\text{YesSharesOwned}_u^\text{Normalized} = 0
+\]
+
+\subsection*{Output}
+
+The final output is a set of normalized market positions:
+
+\[
+\text{NormalizedPositions} = \{ \text{MarketPosition}_u^\text{Normalized} \,|\, u = 1, 2, \dots, m \}
+\]
+
+where each normalized market position is represented as:
+
+\[
```

Comment:
You have a text wrapping error here in the PDF and the normalized market position is unreadable